Core/HLE/sceUsb.cpp

```diff
@@ -122,9 +122,78 @@ const HLEFunction sceUsbstorBoot[] =
 	{0xA55C9E16, 0, "sceUsbstorBootUnregisterNotify"},
 };
 
+const HLEFunction sceUsbCam[] =
+{
+	{ 0x17F7B2FB, 0, "sceUsbCamSetupVideo" },
```

Comment:
Not sure why don't work now.
11:25:863 HLE\sceKernelModule.cpp:686 I[LOAD]: Function (sceUsbCam,17f7b2fb) unresolved, storing for later resolving